src/simulation/monsters/bosses/Sarachnis.ts

```diff
@@ -0,0 +1,67 @@
+import LootTable from '../../../structures/LootTable';
+import SimpleMonster from '../../../structures/SimpleMonster';
+import RareSeedTable from '../../subtables/RareSeedTable';
+import NotedHerbTable from '../../subtables/NotedHerbTable';
+import { GemTable } from '../../subtables/RareDropTable';
+
+const SarachnisTable = new LootTable()
+	.oneIn(192, 'Dragon med helm')
+	.oneIn(384, 'Sarachnis cudgel')
+
+	/* Weapons and armour */
+	.add('Battlestaff', [8, 10], 2)
+	.add('Rune platebody', 1, 2)
+	.add('Rune med helm', 1, 2)
+	.add('Rune 2h sword', 1, 2)
+
+	/* Runes and ammunition */
+	.add('Blood rune', [80, 100], 5)
+	.add('Chaos rune', [175, 200], 5)
+	.add('Cosmic rune', [125, 150], 5)
+	.add('Death rune', [80, 100], 5)
+	.add('Soul rune', [80, 100], 5)
+	.add('Mithril arrow', [450, 600], 2)
+	.add('Mithril bolts', [175, 225], 2)
+
+	/* Seeds */
+	.add(RareSeedTable, 1, 2)
```

Comment:
```suggestion
	.add(RareSeedTable, 3, 2)
```

`There is a 2/100 chance of rolling the rare seed drop table. Three random seeds are dropped at a time. Maple, yew, and papaya tree seeds are separately dropped alone.`

and you need to create additional separate `.add` lines for the maple yew and papaya seeds